code/game/verbs/suicide.dm

```diff
@@ -158,3 +158,7 @@
 		affected.add_autopsy_data(byitem ? "Suicide by [byitem]" : "Suicide", dmgamt)
 
 	updatehealth()
+
+/mob/living/brain/do_suicide()
+	to_chat(viewers(src), "<span class='danger'>[src] is thinking about thinking about thinking about thinking about.... It looks like [p_theyre()] trying to commit suicide.</span>")
```

Comment:
```suggestion
	to_chat(viewers(src), "<span class='danger'>[src] is thinking about thinking about thinking about thinking about.... It looks like [p_theyre()] trying to commit suicide!</span>")
```

Reply:
And should probably use 3 periods instead of 4.

Reply:
Four is correct, actually.  The first 3 are an ellipsis, and the last on is the period at the end of the sentence.  Three is correct mid-sentence, but the end of a sentence should have three plus a period, question mark, or exclamation point.